scan/src/reusable/MobileCard.re

```diff
@@ -35,20 +36,37 @@ let make = (~values, ~idx, ~status=?) => {
            | InfoMobileCard.Messages(_) => `baseline
            | _ => `center
            };
+         let isEmptyString =
+           switch (value) {
+           | InfoMobileCard.Text(text) => text == "" ? true : false
+           | _ => false
+           };
+
```

Comment:
IMO, we can declare a new variant called `Nothing` in `InfoMobileCard.re`
and let's return `React.null` for this case.